### PR TITLE
Experimental filters

### PR DESCRIPTION
**Pull request by chrmod**
Allow users to enable experimental filters which may help with some websites like currently Youtube.com

Experimental filters, by definition are not well tested so may cause breakage, that's why they cannot be enabled by default. 

Screenshot 2024-06-14 at 20 02 22

### CODE REVIEW

File: extension-manifest-v3/src/store/options.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ export const SYNC_OPTIONS = [
   'trackerCount',
   'wtmSerpReport',
   'serpTrackingPrevention',
+  'experimentalCosmeticFilters',
```
Comment (smalluban) [inline extension-manifest-v3/src/store/options.js:32]: I would name this option just `experimental` - as this not mean only cosmetics or filters (at all for DNR version)

Comment (smalluban) [inline extension-manifest-v3/src/store/options.js:32]: I would name this option just `experimental` - as this not mean only cosmetics or filters (at all for DNR version)
You have even a bug here, as this key does not match the below one ;)

File: extension-manifest-v3/src/utils/engines.js [hunk 2]
```diff
@@ -430,4 +432,11 @@ chrome.alarms.onAlarm.addListener((alarm) => {
   }
 });
 
+observe('experimentalFilters', (experimentalFilters) => {
```
Comment (smalluban) [inline extension-manifest-v3/src/utils/engines.js:435]: You are using this in line 42, but this observe is in 435. Please move it close to where the key is used.

**Review by smalluban — CHANGES_REQUESTED** [review]

File: extension-manifest-v3/src/store/options.js (hunk 1 above)
Comment (chrmod) [reply extension-manifest-v3/src/store/options.js]: Prefer to call it `experimentalFilters` as I expect we can have different experiments in near feature

**Review by smalluban — APPROVED** [review]

**Pull request merged**